eppo_client/__init__.py

```diff
@@ -2,15 +2,15 @@
 from eppo_client.client import EppoClient
 from eppo_client.config import Config
 from eppo_client.configuration_requestor import (
-    ExperimentConfigurationDto,
     ExperimentConfigurationRequestor,
 )
 from eppo_client.configuration_store import ConfigurationStore
 from eppo_client.constants import MAX_CACHE_ENTRIES
 from eppo_client.http_client import HttpClient, SdkParams
+from eppo_client.models import Flag
 from eppo_client.read_write_lock import ReadWriteLock
 
-__version__ = "1.3.1"
```

Comment:
I'm proposing to update all SDKs that are based on the UFC to version 3 to make it easier to ensure all SDKs are interoperable. Since our current Java SDK is on version 2.+, we need to jump to 3

Reply:
Yup this is mainly for this one update because the sharding will not be backward compatible with previous versions of the SDKs.